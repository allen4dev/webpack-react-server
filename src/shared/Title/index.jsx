import React from 'react';

import './index.css';

const Title = () => (
  <div className="Title">
    <h1 className="Title-text">This is a dummie title</h1>
  </div>
);

export default Title;
