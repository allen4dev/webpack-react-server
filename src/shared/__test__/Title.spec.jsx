import React from 'react';
import { shallow } from 'enzyme';

import Title from './../Title';

test('Title shoud be render', () => {
  const title = shallow(<Title />);

  expect(title).toMatchSnapshot();
});
