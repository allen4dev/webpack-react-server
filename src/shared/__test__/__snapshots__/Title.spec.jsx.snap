// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title shoud be render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Title />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <h1
        className="Title-text"
      >
        This is a dummie title
      </h1>,
      "className": "Title",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "This is a dummie title",
        "className": "Title-text",
      },
      "ref": null,
      "rendered": "This is a dummie title",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <h1
          className="Title-text"
        >
          This is a dummie title
        </h1>,
        "className": "Title",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "This is a dummie title",
          "className": "Title-text",
        },
        "ref": null,
        "rendered": "This is a dummie title",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
